rails_programming/forms_and_authentication/project_forms.md

```diff
@@ -117,6 +117,7 @@ Send us your solution by editig this [file](https://github.com/TheOdinProject/cu
   <summary> Show Student Solutions </summary>
 
 * Add your solution below this line!
+* [Run After's Solution](https://github.com/powei94/rails_re-former)
```

Comment:
Hey I think you have forgotten to put your name here! Can you fix please?

Reply:
my bad, just fixed it, thank you!